shell.nix

```diff
@@ -1,13 +1,31 @@
 with import <nixpkgs> {};
 
-stdenv.mkDerivation {
-  name = "trezor-user-env";
-  buildInputs = [
-    docker
-    docker-compose
-    xorg.xhost
-    SDL2
-    SDL2_image
-    wget
-  ];
-}
+# the last successful build of nixos-20.09 (stable) as of 2020-10-11
+with import
+  (builtins.fetchTarball {
+    url = "https://github.com/NixOS/nixpkgs/archive/0b8799ecaaf0dc6b4c11583a3c96ca5b40fcfdfb.tar.gz";
+    sha256 = "11m4aig6cv0zi3gbq2xn9by29cfvnsxgzf9qsvz67qr0yq29ryyz";
+  })
+{ };
```

Comment:
@mmilata if we wish to update to latest build of nixos-21.05 (stable) we'll modify the commit to the latest one in `release-21.05` [branch](https://github.com/NixOS/nixpkgs/tree/release-21.05)? Or is it a different one?

Reply:
Preferably in `nixos-21.05`, `release-21.05` is the same but before passing CI. I replace the sha256 with 52 zeroes, then replace it again with the one `nix-shell` prints in the error message. Or you can use `nix-prefetch-url` to get the hash.

Reply:
Good trick :). So I got commit `e9148dc1c30e02aae80cc52f68ceb37b772066f3` and hash `1xs5all93r3fg4ld13s9lmzri0bgq25d9dydb08caqa7pc10f5ki`. So let's try that?

Reply:
Yes, seems to work fine, I am about to commit these changes. Thanks for the help